templates/base/Dockerfile.hbs

```diff
@@ -1,15 +1,13 @@
-FROM node:16
```

Comment:
can we update this to `node:18-slim`?

Reply:
too late for this PR unfortunately. additionally, if the size of the image is a concern then they way it's built as one layer should also be looked at